RevenueCatUI/Views/LoadingPaywallView.swift

```diff
@@ -44,7 +44,8 @@ struct LoadingPaywallView: View {
             fonts: DefaultPaywallFontProvider(),
             displayCloseButton: self.displayCloseButton,
             introEligibility: Self.introEligibility,
-            purchaseHandler: Self.purchaseHandler
+            purchaseHandler: Self.purchaseHandler,
+            locale: Locale.current
```

Comment:
Should we be defaulting to `Locale.preferredLocales.first ?? Locale.current`? (access restrictions aside.)

Reply:
Ah, probably! The loading view is all skeleton stuff anyway so there isn't _actually_ any text anywhere but that does feel better 😇